source/opt/eliminate_dead_input_components_pass.h

```diff
@@ -28,7 +28,8 @@ namespace opt {
 // See optimizer.hpp for documentation.
 class EliminateDeadInputComponentsPass : public Pass {
  public:
-  explicit EliminateDeadInputComponentsPass() {}
+  explicit EliminateDeadInputComponentsPass(bool output_instead = false)
```

Comment:
If you rename the pass as spencer suggested, I'd be in favor to not default the value to false, but to make it mandatory.

Reply:
Agreed